src/IO/AbstractIOHandlerHelper.cpp

```diff
@@ -75,6 +86,7 @@ std::shared_ptr<AbstractIOHandler> createIOHandler<json::TracingJSON>(
         return constructIOHandler<ParallelHDF5IOHandler, openPMD_HAVE_HDF5>(
             "HDF5", path, access, comm, std::move(options));
     case Format::ADIOS1:
+        std::cerr << adios1Deprecation << std::endl;
```

Comment:
I am not sure if this is the cleanest way to warn users, because it cannot be suppressed. This might also be a problem in MPI-parallel contexts.

Maybe we can add an env var to turn it off or something.